packages/machine-extractor/src/identifiers.ts

```diff
@@ -150,7 +150,7 @@ export const objectExpressionWithDeepPath = <Result>(
 const getRootIdentifierOfDeepMemberExpression = (
   deepMemberExpression: DeepMemberExpression | undefined,
 ): t.Identifier | undefined => {
-  if (!deepMemberExpressionToPath) return undefined;
+  if (!deepMemberExpression) return undefined;
```

Comment:
😅

Reply:
Yeah, this one was fun. It caused a `Maximum call stack size exceeded` because it just kept checking if the function  `deepMemberExpressionToPath` was still defined 😅